app/flags/flag.tsx

```diff
@@ -74,10 +74,29 @@ const initFromSearchParams = (locationSearch: string) => {
   }
 };
 
+const isVercelPreviewHost = (host: string) => {
+  return !!/visualization\-tool.*ixt1\.vercel\.app/.exec(host);
+};
```

Comment:
Nit: Could maybe be improved with basing on Vercel's env variables (e.g. checking if VERCEL_URL is not undefined)? https://vercel.com/docs/projects/environment-variables/system-environment-variables#system-environment-variables

Reply:
Ah good idea, but then I would need to pass it through the env so that this is built and sent to the client. Might be overkill here.